build/driver.mk

```diff
@@ -87,7 +87,7 @@ XCTRACER_SOURCES = \
 
 THERMALEXPRESS_SOURCES = \
 	$(DRIVER_SRC_DIR)/ThermalExpress/Driver.cpp
- 
+
```

Comment:
This is an example of a line where we had a mistakenly added trailing space, and you removed it in this unrelated commit. This is confusing to read, and this is why there shouldn't be trailing whitespace. (See below for examples of trailing whitespace you added.)